packages/labs/ssr/package.json

```
@@ -228,14 +228,16 @@
     "@lit-labs/ssr-dom-shim": "^1.2.0",
     "@lit/reactive-element": "^2.0.4",
     "@parse5/tools": "^0.3.0",
-    "@types/node": "^16.0.0",
     "enhanced-resolve": "^5.10.0",
     "lit": "^3.1.2",
     "lit-element": "^4.0.4",
     "lit-html": "^3.1.2",
     "node-fetch": "^3.2.8",
     "parse5": "^7.1.1"
   },
+  "peerDependencies": {
+    "@types/node": ">=18.0.0 <25.0.0"
```

I think it might be a bit early to include Node24, as development on it as not started yet.
```suggestion
    "@types/node": ">=18.0.0 <24.0.0"
```

---

While I agree, this will force us to update it every 6 months, for an API (`import {Readable} from 'stream'`) that is unlikely to change.
From my understanding there is not yet an automation to update dependencies, so someone will continuously have to remember to update the range.
This range will give us time until the end of next year, while Node.js 24 will be released in April (although not yet as LTS).